Jenkinsfile

```diff
@@ -18,7 +15,7 @@ properties([
     pipelineTriggers([pollSCM('H/15 * * * *')])
 ])
 
-node('docker') {
+node('sdk && docker') {
```

Comment:
Okay, we're already using Gentoo-style architecture labels, so `amd64 && docker` would work here for us.  I also think that label would better reflect the problem that is trying to be solved.

I will comment similarly on that other PR.